garden_ai/client.py

```diff
@@ -1,14 +1,14 @@
+# mypy: disable-error-code="import"
```

Comment:
this disables mypy import checking for this file, because black wouldn't let me keep the `# type: ignore` comment on the same line as the `from mlflow.tracking.request_header.registry import ...`